pkg/k8shandler/defaults.go

```diff
@@ -8,6 +8,9 @@ var (
 	defaultEsMemory     resource.Quantity = resource.MustParse("16Gi")
 	defaultEsCpuRequest resource.Quantity = resource.MustParse("1")
 
+	defaultEsProxyMemory     resource.Quantity = resource.MustParse("256Mi")
+	defaultEsProxyCpuRequest resource.Quantity = resource.MustParse("100m")
```

Comment:
Why is this mandatory? The default in 4.5 was `64Mi`. 4.6 and 4.7 only got the bump to `256Mi` for now.

Reply:
this was done based on your PR to change the defaults? are you sure we dont need your PR to be backported?

Reply:
updated